src/main/java/formflow/library/FileController.java

```diff
@@ -181,4 +213,117 @@ RedirectView delete(
       return new RedirectView("/error");
     }
   }
+
+  /**
+   * @param httpSession  - The current httpsession
+   * @param submissionId - The submissionId of the file to be downloaded
+   * @param fileId       - The UUID of the file to be downloaded.
+   * @return ON SUCCESS: A ResponseEntity with a status of ok, a header, the contentType, the content size, and a responseBody
+   * that includes the file.
```

Comment:
`responseBody` is the name of the variable, which requires someone to read the code to figure out what it actually us.  You could just call  say "response body".